app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -2,6 +2,8 @@ package org.fossasia.openevent.general.event
 
 import android.arch.lifecycle.MutableLiveData
 import android.arch.lifecycle.ViewModel
+import android.content.Intent
+import android.net.Uri
```

Comment:
No Android

Reply:
@iamareebjamal If I move the intents and even the uri parsing to the normal fragment, won't the logic be too less to handle in the view model?

Reply:
It will be the testable part. Other stuff like creating intents can be handled by Android containers